src/app/visual/inputs/inputs-demo.component.html

```diff
@@ -1,20 +1,131 @@
 <div style="width: 400px">
   <div id="group1">
-    <input type="button" class="sky-form-control" value="Click me">
-    <input type="checkbox" name="vehicle1" value="Bike" class="sky-form-control"> I have a bike<br>
-    <input type="checkbox" name="vehicle2" value="Car" class="sky-form-control"> I have a car<br>
-    <input type="checkbox" name="vehicle3" value="Boat" checked class="sky-form-control"> I have a boat<br>
-    Select your favorite color: <input type="color" name="favcolor" class="sky-form-control"><br>
-    Birthday: <input type="date" name="bday" class="sky-form-control"><br>
-    Birthday (date and time): <input type="datetime-local" name="bdaytime" class="sky-form-control"><br>
-    E-mail: <input type="email" name="emailaddress" class="sky-form-control"><br>
-    Select a file: <input type="file" name="myFile" class="sky-form-control"><br>
-    <input type="hidden" id="custId" name="custId" value="3487" class="sky-form-control"><br>
-    <input type="image" style="width:100px; height:100px" [src]="imageData" alt="Submit" class="sky-form-control"><br>
-    Birthday (month and year): <input type="month" name="bdaymonth" class="sky-form-control"><br>
-    Quantity (between 1 and 5): <input type="number" name="quantity" min="1" max="5" class="sky-form-control"><br>
+    <button
+      (click)="onDisableToggleClick()"
```

Comment:
☝️ Added a button to the visual demo to easily demonstrate what disabled looks like.